amazonsqs/amazonsqs_connector.bal

```diff
@@ -0,0 +1,193 @@
+    int attributeNumber = 1;
+    foreach var (k, v) in attributes {
+        payload = payload + "&Attribute." + attributeNumber + ".Name=" + k;
+        payload = payload + "&Attribute." + attributeNumber + ".Value=" + v;
+        attributeNumber = attributeNumber + 1;
+    }
+
+    http:Request request = check generatePOSTRequest(self.accessKey, self.secretKey, self.host, amzTarget, endpoint, self.region, payload);
+    var httpResponse = self.clientEp->post(endpoint, request);
+    json|error response = handleResponse(httpResponse);
+    if(response is json){
+        return jsonToCreatedQueueUrl(response);
+    } else {
+        return response;
+    }
+
+}
+
+# Send a new message to a SQS queue
+#
+# + messageBody - Message body string to be sent 
+# + queueResourcePath - Resource path to the queue from the host address 
+# + attributes - Non-mandatory parameters for sending a message 
+# + return - Details of the sent message
+public remote function Client.sendMessage(string messageBody, string queueResourcePath, map<string> attributes) returns OutboundMessage|error {
+
+    string amzTarget = "AmazonSQSv20121105.SendMessage";
+    string payload =  "";
+    payload = payload + "Action=SendMessage";
+    payload = payload + "&MessageBody=" + check http:encode(messageBody, "UTF-8");
+
+    int attributeNumber = 1;
+    foreach var (k, v) in attributes {
+        payload = payload + "&" + k + "=" + v;
+        attributeNumber = attributeNumber + 1;
+    }
+
+    http:Request request = check generatePOSTRequest(self.accessKey, self.secretKey, self.host, amzTarget, queueResourcePath, self.region, payload);
+    var httpResponse = self.clientEp->post(queueResourcePath, request);
+    json|error response = handleResponse(httpResponse);
+    if(response is json){
+        return jsonToOutboundMessage(response);
+    } else {
+        return response;
+    }
+
+}
+
+# Description
+#
+# + queueResourcePath - Resource path to the queue from the host address
+# + attributes - Non-mandatory parameters for receiving a message
+# + return - Details of the received message
+public remote function Client.receiveMessage(string queueResourcePath, map<string> attributes) returns InboundMessage[]|error {
+
+    string amzTarget = "AmazonSQSv20121105.ReceiveMessage";
+    string payload =  "";
+    payload = payload + "&Action=ReceiveMessage";
+    int attributeNumber = 1;
+    foreach var (k, v) in attributes {
+        payload = payload + "&" + k + "=" + v;
+        attributeNumber = attributeNumber + 1;
+    }
+
+    http:Request request = check generatePOSTRequest(self.accessKey, self.secretKey, self.host, amzTarget, queueResourcePath, self.region, payload);
+    var httpResponse = self.clientEp->post(queueResourcePath, request);
+    json|error response = handleResponse(httpResponse);
+    if(response is json){
+        return jsonToInboundMessages(response);
+    } else {
+        return response;
+    }
+    
+}
+
+# Description
```

Comment:
Documentation?  This will be shown in Ballerina Central

Reply:
Added